tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -299,6 +278,35 @@ <h3>No precision–recall curve data was found.</h3>
         const defaultDisplayName = tag.replace(/\/pr_curves$/, '');
         return tf_utils.aggregateTagInfo(runToTagInfo, defaultDisplayName);
       },
+      _createDataChangeCallback(tag) {
+        return function onDataChange(runToData) {
+          this.set(
+            '_tagToRunToData',
+            Object.assign({}, this._tagToRunToData, {[tag]: runToData})
```

Comment:
`{...this._tagToRunToData, [tag]: runToData}` reads pretty well :)

Reply:
Huh, I thought for some reason that we only supported array-spread and
not object-spread; there are no existing matches for `{...` in HTML
files. But indeed this seems to work fine. Done; thanks.